### PR TITLE
Move the dataplotly sample file out of training_data.gpkg

### PR DESCRIPTION
**Pull request by DelazJ**
The prepare-data lesson instructs the user to create the training-data.gpkg file, populated with OSM data. For someone who followed that lesson, there's no way he has the sample_points layer in his DB.
Also apply some formatting.
See also https://github.com/qgis/QGIS-Training-Data/pull/19

### CODE REVIEW

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst [hunk 1]
```diff
@@ -167,32 +167,35 @@ Take a look at some example and try to explore the query language.
 |basic| |FA| The DataPlotly Plugin
 -------------------------------------------------------------------------------
 
-DataPlotly plugin allows you to create `D3 <https://d3js.org/>`_ plots of vector
+**DataPlotly** plugin allows you to create `D3 <https://d3js.org/>`_ plots of vector
```
Comment (havatv) [inline source/docs/training_manual/qgis_plugins/plugin_examples.rst]:
```suggestion
The **DataPlotly** plugin allows you to create `D3 <https://d3js.org/>`_ plots of vector
```

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst [hunk 2]
```diff
@@ -167,32 +167,35 @@ Take a look at some example and try to explore the query language.
...
-   :align: center
+   .. image:: img/dataplotly_setup.png
+      :align: center
 
-If you want you can change the colors, the marker type, the transparency and
-many other settings: just try to change some parameters to create the plot as the
-following picture. Once you set up all the parameters click on the
-:guilabel:`Create Plot` button to create the plot:
+#. If you want you can change the colors, the marker type, the transparency and
+   many other settings: just try to change some parameters to create the plot
```
Comment (havatv) [inline source/docs/training_manual/qgis_plugins/plugin_examples.rst]:
```suggestion
   many other settings: try to change some parameters to create the plot
```

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst [hunk 3]
```diff
@@ -167,32 +167,35 @@ Take a look at some example and try to explore the query language.
...
+   as the following picture.
```
Comment (havatv) [inline source/docs/training_manual/qgis_plugins/plugin_examples.rst]:
```suggestion
   below.
```

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst [hunk 4]
```diff
@@ -167,32 +167,35 @@ Take a look at some example and try to explore the query language.
...
+#. Once you set up all the parameters click on the :guilabel:`Create Plot`
+   button to create the plot.
 
-.. image:: img/dataplotly_scatterplot.png
-   :align: center
+   .. image:: img/dataplotly_scatterplot.png
+      :align: center
```
Comment (havatv) [inline source/docs/training_manual/qgis_plugins/plugin_examples.rst]: Perhaps this image should be moved up one item (to where it is referenced from).

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst [hunk 5]
```diff
@@ -167,32 +167,35 @@ Take a look at some example and try to explore the query language.
...
 
-If you want you can change the colors, the marker type, the transparency and
-many other settings: just try to change some parameters to create the plot as the
-following picture. Once you set up all the parameters click on the
-:guilabel:`Create Plot` button to create the plot:
+#. If you want you can change the colors, the marker type, the transparency and
+   many other settings: just try to change some parameters to create the plot
+   as the following picture.
+#. Once you set up all the parameters click on the :guilabel:`Create Plot`
```
Comment (havatv) [inline source/docs/training_manual/qgis_plugins/plugin_examples.rst]:
```suggestion
#. Once you have set all the parameters, click on the :guilabel:`Create Plot`
```

File: source/docs/training_manual/qgis_plugins/plugin_examples.rst (hunk 4 above)
Comment (DelazJ) [reply source/docs/training_manual/qgis_plugins/plugin_examples.rst]: I'm not sure I follow. I think this screenshot is obtained AFTER you press the "create plot" button (or a minima I don't see any "create plot" button on it that would justify a move up).

Comment (havatv) [reply source/docs/training_manual/qgis_plugins/plugin_examples.rst]: My point was that the image is referenced in lines 192-193.  I don't think it is a problem that it is placed above line 194.

**Pull request merged**